demo-app/yarn.lock

```diff
@@ -9732,10 +9732,9 @@ react-bootstrap@^1.5.2:
     uncontrollable "^7.2.1"
     warning "^4.0.3"
 
-react-clip-path@^0.0.3:
-  version "0.0.3"
-  resolved "https://registry.yarnpkg.com/react-clip-path/-/react-clip-path-0.0.3.tgz#88aabf4c6ac55ba8875551a0a5a1a8fb918e0973"
-  integrity sha512-GbdsHU4o1YygJLpQafy6g+U8+6STRW+SlePCFZiFuS/j041X/Px3QdyjE92Lr2y2mxDzS1bId5yi64Q/W2VW8A==
+"react-clip-path@file:react-clip-path-0.0.4.tgz":
```

Comment:
Do a clean yarn install and then commit the yarn.lock file. It still points to the local file

Reply:
Done